addons/website_slides/controllers/main.py

```diff
@@ -122,6 +122,22 @@ def _get_slide_detail(self, slide):
     def _get_slide_quiz_info(self, slide, quiz_done=False):
         return slide._compute_quiz_info(request.env.user.partner_id, quiz_done=quiz_done)[slide.id]
 
+    def _get_slide_quiz_data(self, slide):
```

Comment:
Maybe we could rename _get_slide_quiz_info to _get_slide_quiz_partner_info to indicate it returns data related to the partner, and quiz_data returns overall data related to the slide question. Otherwise difference between info and data is unclear :) .